tests/foreman/installer/test_installer.py

```diff
@@ -1627,19 +1627,10 @@ def test_positive_check_installer_hammer_ping(target_sat):
     :CaseLevel: System
     """
     # check status reported by hammer ping command
-    result = target_sat.execute('hammer ping')
-    test_result = {}
-    service = None
-    for line in result.stdout.strip().replace(' ', '').split('\n'):
-        if line.split(':')[0] not in ('Status', 'ServerResponse', 'message'):
-            service = line.split(':')[0]
-            test_result[service] = {}
-        else:
-            key, value = line.split(":", 1)
-            test_result[service][key] = value
-
-    not_ok = {svc: result for svc, result in test_result.items() if result['Status'] != 'ok'}
-    assert not not_ok
+    result = target_sat.cli.Base.ping()
+    for line in result.split('\n'):
+        if 'Status' in line:
+            assert 'ok' in line
```

Comment:
This is a weaker assertion. For example, when I stop the foreman services, I get this output and the assertion won't fail for it:

```
Error: Failed to open TCP connection to <FQDN>:443 (Connection refused - connect(2) for "<FQDN>" port 443)
```